coffee_core/src/coffee.rs

```diff
@@ -330,6 +330,15 @@ impl PluginManager for CoffeeManager {
         );
         Err(err)
     }
+
+    async fn search(&mut self, plugin: &str) -> Result<Value, CoffeeError> {
+        for repo in &self.repos {
+            if let Some(_plugin) = repo.get_plugin_by_name(plugin) {
+                return Ok(json!({ "search": repo.name() }));
```

Comment:
```suggestion
                return Ok(json!({ "search": repo.name() }));
```

I would like put a more complex json here, like 

```json
{
   "repository":  "<link>",
  "plugin": {
          "conf": {
            "plugin": {
              "dependencies": null,
              "deprecated": null,
              "install": "make dep\nmake coffee\n",
              "lang": "go",
              "main": "go-lnmetrics",
              "name": "lnmetrics",
              "version": "0.0.6-rc2"
            }
          },
          "lang": "Go",
          "name": "lnmetrics",
          "path": "/home/vincent/.coffee/repositories/lightningd/go-lnmetrics.reporter",
          "root_path": "/home/vincent/.coffee/repositories/lightningd/go-lnmetrics.reporter"
        }
}
```

Inside the plugin field, we store the same information that are inside the `coffee list` command

Reply:
Alright. So this should be the returned JSON but the messages should stay the same?
if found: ✓ found plugin helpme in remote repository https://github.com/lightningd/plugins

Reply:
we will fix the message later, for now any kind of message is fine, so yes let keep this as it is